server/controllers/post/create.js

```diff
@@ -0,0 +1,52 @@
+// modules
+const uuid = require('uuid/v1');
+
+const database = require('../../database')
+
+exports.create = (req, res, next) => {
+	const postTitle = req.body.title;
+	const postSlug = req.body.slug;
```

Comment:
I still need to write the logic to convert title to slug.

Any thoughts on that?

Reply:
First thoughts:
1. Make all chars lowercase
1. Replace space with `-`
1. Append the random ID

Seems enough, I think.

Reply:
That's good, but, it's still not enough.

* Remove special characters
* Down case all characters
* Replace spaces with `-`

* Append random ID _(it's obvious so no need to add in the list)_

Reply:
So, how should we do it 😅

Reply:
Okay.
`req.body.title.replace(/[^\w\s]/gi, '').toLowerCase().replace(' ', '-') + randomId`

Try something like that.

Reply:
The problem with this regex query is that it will not recognise any double or triple spaces.

For example:

```
k          k
```

will be converted to `-------`